backend/substrapp/tasks/tasks.py

```diff
@@ -720,6 +722,18 @@ def do_task(channel_name, subtuple, tuple_type):
     models = save_models(subtuple_directory, tuple_type, subtuple['key'])  # Can be empty if testtuple
     result = extract_result_from_models(tuple_type, models)  # Can be empty if testtuple
 
+    # Handle local volume
+    if compute_plan_key is not None:
+        local_path = path.join(subtuple_directory, 'local')
+
+        # Get CP local folder from local pvc
+        local_path_in_pvc = get_local_folder(compute_plan_key)
+
+        # Copy content of local folder from subtuple directory to local folder in pvc
+        # to fetch modification of local folder content
+        distutils.dir_util.copy_tree(local_path, local_path_in_pvc)
+        logger.info(f'Content from sandbox {local_path} has been copied to pvc {local_path_in_pvc}')
+
```

Comment:
What about extracting the contents of this whole block into a function `_commit_local_folder`?

The preparation block (in `prepare volumes`) could be extracted into a `_prepare_local_folder` function

Reply:
I don't know if it's necessary to do so.
For instance, in `prepare_volumes`, it's more explicit to see that the `local volume` part is doing modification over the model dictionary.
For the do task, it 4 lines of code. But I can extract the contents to make them function if other find it better !